src/reporter.ts

```diff
@@ -635,4 +653,17 @@ export class RPReporter implements Reporter {
   printsToStdio(): boolean {
     return false;
   }
+
+  #extractTagsFromTitle(title: string): string {
+    return (
+      title
+        .match(/@\w+\s*/g)
+        ?.join('')
+        .trim() || ''
+    );
+  }
+
+  #getAttributesFromTitle(title: string): Attribute[] {
+    return title.match(/@\w+\s*/g)?.map((tag) => ({ key: 'tag', value: tag.trim() })) || [];
```

Comment:
As we discussed earlier, the attributes can be `@key:value` or just `@value`, so please consider the two cases here.
In case of only `value` presented, `key` should be empty.